client_plugin/Makefile

```diff
@@ -397,6 +399,10 @@ test-e2e-runonce:
 	$(log_target)
 	$(GO) test -v -timeout 40m -tags runonce $(E2E_Tests)
 
+test-e2e-runonce-windows:
+	$(log_target)
+	$(GO) test -v -timeout 40m -tags "runoncewin winutil" $(E2E_Tests)
```

Comment:
It's required so that the test picks up the right [`ssh`](https://github.com/vmware/docker-volume-vsphere/blob/96e104ef6ae09ae1240eefb5c02170a2b805f15a/tests/utils/ssh/ssh_win.go#L17) and [`testparams`](https://github.com/venilnoronha/docker-volume-vsphere/blob/580b3f03ee65698f15a67eca1ff67e4f2cac31a0/tests/utils/inputparams/testparams_win.go#L17) utils.